CHANGELOG.md

```diff
@@ -61,20 +62,21 @@
 * **Breaking change:** instead of
 
   ```js
-<AppContainer component={App} props={{ prop: val }} />
-```
+  <AppContainer component={App} props={{ prop: val }} />

```

Comment:
But it never accepts component. V4 does not change behavior of AppContainer at all.

Reply:
Nevermind.. :0

Reply:
This is old changelog reformated, nothing important.